### PR TITLE
feat: Pbi 19 improve api monitor stats frontend

### LINKED ISSUE #50: Improve api monitor dashboard statistics

### PR DESCRIPTION
**Pull request by ferdifdi**
### Describe your changes
- introduce yellow bar (dashboard and detail)
- highlight create new api monitor when user have no api monitor
- last checked
- color indicator description

image
image
image

### Backlog name
PBI-19-improve_api_monitor_stats-frontend

### Checklist
- [X] I have performed a self-review of my code
- [X] Code successfully run on local
- [X] Feature / changes tested on local
- [X] No failing unit test
- [ ] Passed UAT Test
- [ ] Sonarqube tested 
- [ ] Required any changes to environment variable

### CODE REVIEW

File: src/routes/view_list_monitor/index.js [hunk 1]
```diff
@@ -59,19 +66,45 @@ const ViewListMonitor = () => {
 					<button type="button" class="btn btn-success">Create New</button>
 				</Link>
 			</div>
-			
+			<br/>
+			{(monitor.length != 0)?
+				<div>
+					<p>Last checked: {dateTime}</p>
```
Comment (andrew-susanto) [inline src/routes/view_list_monitor/index.js]: last checkednya per monitor ferr di API nya ada fieldnya, di .last_result.execution_time

**Review by luthfifahlevi — COMMENTED** [review]
sepertinya blm dibuat yellow-bar di dalam chart success_rate_percentage?

**Review by andrew-susanto — APPROVED** [review]

File: src/routes/view_list_monitor/index.js [hunk 2]
```diff
@@ -116,7 +110,8 @@ const ViewListMonitor = () => {
 							{val.success_rate_history.map((history, idx)=>(
 								<SuccessRate success={history.success} failed={history.failed} key={idx} /> 						
 							))}	
-							</div>											
+							</div>
+							<p>Last checked: {val.last_result?.execution_time}</p>
```
Comment (luthfifahlevi) [inline src/routes/view_list_monitor/index.js]: Seingatku format yg di execution_time masih blm readable, jdi perlu direformat dlu. Klo mau gampang, mungkin boleh tiru yg di error log list aja pake toLocateDateString

Reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString

Comment (luthfifahlevi) [reply src/routes/view_list_monitor/index.js]: Oh ya, mungkin diupdate jg deskripsi PRnya dengan screenshot yg baru hehe titip ya fer, thank you 🙏

Comment (andrew-susanto) [reply src/routes/view_list_monitor/index.js]: ini kayaknya pake moment js aja fer

caranya 
- jalanin `npm install moment`
- lalu tulis `import moment from "moment"`
- lalu code yg val... diatas diganti jadi `{val.last_result ? moment(val.last_result.execution_time).fromNow() : "-"}`

**Review by luthfifahlevi — APPROVED** [review]
nice, thank you

**Pull request merged**